index.bs

```diff
@@ -735,6 +737,12 @@ When this method is invoked, the user agent MUST execute the following algorithm
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
             and [=set/remove=] |authenticator| from |issuedRequests|.
 
+        :   If the <code>|options|.{{MakePublicKeyCredentialOptions/signal}}</code> is [=present=] and and the 
```

Comment:
Assuming you keep Jeff's suggestion to put the AbortSignal in CredentialCreationOptions instead of MakePublicKeyCredentialOptions (which I agree with, as mentioned in https://github.com/w3c/webappsec-credential-management/pull/104), you'll need to pull the signal out before step 2 where you overwrite |options|.

Reply:
AbortSignal was put in credman's CredentialCreationOptions via credman PR w3c/webappsec-credential-management#110